src/lib/components/workflow/workflows-summary-configurable-table.svelte

```diff
@@ -66,78 +66,81 @@
       (id) => id.workflowId === workflow.id && id.runId === workflow.runId,
     );
   };
+
+  $: onFetch = () => fetchPaginatedWorkflows(namespace, $workflowsQuery);
 </script>
 
-<PaginatedTable
-  aria-label={translate('common.workflows')}
-  perPageLabel={translate('common.per-page')}
-  nextPageButtonLabel={translate('common.next-page')}
-  previousPageButtonLabel={translate('common.previous-page')}
-  pageButtonLabel={(page) => translate('common.go-to-page', { page })}
-  loading={$loading}
-  updating={$updating}
-  items={$workflows}
-  let:visibleItems
->
-  <caption class="sr-only" slot="caption">
-    {translate('common.workflows')}
-  </caption>
-  <TableHeaderRow
-    columnsCount={columns.length}
-    {empty}
-    slot="headers"
+{#key [namespace, $workflowsQuery, $refresh]}
+  <PaginatedTable
+    total={$workflowCount.count}
+    {onFetch}
     let:visibleItems
-    workflows={visibleItems}
+    aria-label={translate('common.workflows')}
+    pageSizeSelectLabel={translate('common.per-page')}
```

Comment:
Noticed when changing the pageSize (and there are new Workflows) the total count gets a bit off ⬇️ 

![Screenshot 2025-05-07 at 3 59 06 PM](https://github.com/user-attachments/assets/d745eb9d-e63e-4023-9b4c-143e4be3cf59)

Reply:
Good catch. A couple of things here:

1. I wasn't refretching the workflow counts when the perPage size changed. Fixed that.
2. There can be a slight difference between the workflow counts total and the actual number returned in ListWorkflows due to timing. So the last page (which can be the first page) might have an endIndex (total number) different than the total count. So I adjusted it to use endIndex instead of total in that situation so it always used the stores total instead of the total prop if it's different.